docs/_articles/en/codebuilder/cb-setup.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Set Up Code Builder
+lang: en
+---
+
+## Install the Code Builder Managed Package
+
+Salesforce Code Builder is a second-generation (2GP) managed package that you can install in any supported Salesforce org edition.
```

Comment:
Seems appropriate to list them here. And any permissions required to install and configure. Can you find out if "Download AppExchange Packages" (which is huge - give you a ton of add-on privileges) or something less powerful is required?

Reply:
Based on the fact that you're sending them to AppExchange, vs. how we deliver DevOps Center, this user permission is probably required.

Reply:
Also this begs some other questions for me. I was thinking that every Dev had to go install the package...but it sounds like an admin installs it in a prod org, and then adds dev users to that org (now we need to discuss required Salesforce licenses - and if they can use the Salesforce Limited Access License), and then they can assign permission sets to those users. But it's likely those dev users won't have accounts in the org unless it's also their Dev Hub.

Reply:
You are correct that only user with Download AppExchange Packages permission can install this 2GP package. it can be installed on any org edition though DE is probably we the "safest". I had a version of the setup that was DE org focused but along the way we decided to keep it general.

Reply:
Do they need the Salesforce Limited Access License to create scratch orgs? They aren't creating those. Here they just launch Code Builder and then connect to the orgs they want to work in. Let me know if that's not clear.